### PR TITLE
Add button to awareness toast; broaden eligibility

### LINKED ISSUE #2261: Multiple-facet experiment for awareness
We want to try the video, docs, etc. as different facets of the awareness experiment.

### PR DESCRIPTION
**Pull request by bwateratmsft**
Fixes #2261

### CODE REVIEW

File: src/telemetry/surveys/awareness.ts [hunk 1]
```diff
@@ -7,14 +7,17 @@ import { ext } from '../../extensionVariables';
...
+const minimumOverallSessions = 2;
 const maximumNotEditOnlySessions = 0;
 
 export const awareness: Survey = {
     id: 'aware',
-    url: 'https://aka.ms/AA8lh3s',
     prompt: localize('vscode-docker.survey.aware.prompt', 'The Docker extension is always adding new features. Do you want to learn more about them?'),
-    buttons: [localize('vscode-docker.survey.aware.button.learnMore', 'Learn more'), localize('vscode-docker.survey.aware.button.never', 'Don\'t show again')],
+    buttons: new Map<string, string | undefined>([
+        [localize('vscode-docker.survey.aware.button.watch', 'Watch a video'), 'https://aka.ms/AA8lh3s'],
```
Comment (bwateratmsft) [inline src/telemetry/surveys/awareness.ts:17]: TODO: strings, URLs

Comment (bwateratmsft) [reply src/telemetry/surveys/awareness.ts:17]: @BigMorty for this

Comment (BigMorty) [reply src/telemetry/surveys/awareness.ts:17]: https://aka.ms/AA9j59w is the link for "Read documentation" button.

Comment (bwateratmsft) [reply src/telemetry/surveys/awareness.ts:17]: Agreed on "Watch a video" and "Read docs", looks like this:
image

**Review by karolz-ms — APPROVED** [review]

**Pull request merged**